src/backend/distributed/citus--7.1-2--7.1-3.sql

```diff
@@ -0,0 +1,17 @@
+/* citus--7.1-2--7.1-3 */
+
+CREATE TABLE citus.pg_dist_node_metadata(
+    metadata jsonb NOT NULL
+);
+ALTER TABLE citus.pg_dist_node_metadata SET SCHEMA pg_catalog;
+GRANT SELECT ON pg_catalog.pg_dist_node_metadata TO public;
+
+CREATE FUNCTION pg_catalog.citus_server_id()
```

Comment:
Would be kind of nice to rename this to something like citus_uuid() since it a) seems generally useful for users that don't have uuid-ossp installed, b) doesn't do a good job of returning the current server ID.

Reply:
Hm, I kind of suggested this name to denote it as something internal-only (and it doesn't say _current_ server id, it just generates one :-P). But I see your point and could be swayed.